plot_fits.py

```diff
@@ -291,12 +291,19 @@ def _parser():
                         'or both.')
     parser.add_argument('--ftype', help='Select which type the fits file is',
                         choices=['ARES', 'CRIRES'], default='ARES')
+    parser.add_argument('--fitsext', help='Select Crires extention',
+                        choices=['0', '1', '2', '3', '4'], default='0')
```

Comment:
The default of 0 returns primary header, and works when there are no extensions. I have also added the fitsext to the getdata call for ARES but with the default of 0 it should still work.  Returns same data as extension 1 with the CRIRES data. Integers are possible with trade offs, see below.